community/modules/compute/schedmd-slurm-gcp-v5-node-group/source_image_logic.tf

```diff
@@ -15,12 +15,47 @@
  */
 
 locals {
-  source_image         = lookup(var.instance_image, "name", "")
-  source_image_family  = lookup(var.instance_image, "family", "")
-  source_image_project = lookup(var.instance_image, "project", "")
-  source_image_project_normalized = (
-    local.source_image != "" || length(regexall("/", local.source_image_project)) > 0
-    ? local.source_image_project
-    : "projects/${local.source_image_project}/global/images/family"
+  # Currently supported images and projects
+  known_project_families = {
+    "schedmd-slurm-public" = ["slurm-gcp-5-7-debian-11", "slurm-gcp-5-7-hpc-rocky-linux-8", "slurm-gcp-5-7-ubuntu-2004-lts",
```

Comment:
Let's have the formatting align with rest of Toolkit choices. Drop the quotation marks around the key (it can only be a string in Terraform) and separate each value by line. e.g.

```hcl
known_project_families = {
  schedmd-slurm-public = [
    "slurm-gcp-5-7-debian-11",
    ...
  ]
}
```

Reply:
Resolved.